lib/query.js

```diff
@@ -2366,7 +2382,7 @@ Query.prototype._find = async function _find() {
   }
 
   if (!mongooseOptions.populate) {
-    const versionKey = _this.schema.options.versionKey;
+    const versionKey = this.schema.options.versionKey;
```

Comment:
I dont exactly know why this change was made, but i think in this function `_this` can be completely removed and directly use `this`.

Reply:
Fixed in a41b821